src/browser-frontend/reducers/pages-tabbar-reducers.js

```diff
@@ -19,6 +19,11 @@ function setTabState(state, { payload: { pageId, tabState } }) {
   return state.setIn(['ui', 'pages', 'visuals', pageId, 'tabState'], tabState);
 }
 
+function changeOptionalClass(state, { payload: { parentId, optionalClass } }) {
```

Comment:
Rename `parentId` to `pageId`, since that is what the "optionalClass" field is changed on. Otherwise, one might assume that this reducer changes the parent tab's UI state, instead of a specific one.